### DISCUSSION TITLE
Date Timer picker select date by plain js

### DISCUSSION THREAD

**Discussion by MrViSiOn** [vuetifyjs/vuetify#14315] [Q&A]
Hi!

I'm getting a little bit of frustation trying to change a value of a input text with vuejs with v-time-picker.

Vuejs never register new value:

![image](https://user-images.githubusercontent.com/67826/138608078-3462cb47-4fc8-4bcc-815d-3c4b9b703645.png)

I have this field that dispatch date and time picker.

![image](https://user-images.githubusercontent.com/67826/138608094-b3274f25-0f72-4685-9236-5d758343d5fe.png)

I executed a lot os snippets but none works, none register value I want...

`var $input = $('#air_boardingDate').val('2021-11-12 11:00');
var e = document.createEvent('HTMLEvents');
e.initEvent('change', true, true);
$input[0].dispatchEvent(e);`

I need this kind of hack because I need to fill up 50 times a form for 50 passengers I am responsible of their travel and I want to automate the submission.

Thank you and sorry if this is not the place for writting this kind of things

EDIT: Form I want to automate is https://app.euplf.eu/#/form

**Comment by ghost** [comment]
what do you exactly mean by none register the value I want? is the value returned by the time picker a different format? Is the value returned by the time picker not being recognized by vuejs? What is your exact issue.
Thanks

**Reply by MrViSiOn** [reply]
The field is marked as non-valid, like there is no value contained/registered/setted.

I want to automate the date selection by triggering clicks by plain JS and I can not get it.

Thank you for answering ;)

**Reply by ghost** [reply]
Could you share your code, does the input have a model? What happens if you log the value?

**Comment by ghost** [comment]
I just reread your question like 5 times, if you need to preset the value of the field. Your string is invalid as a datetime, time needs to be prepended with a T, so your string would become '2021-11-12T11:00'

**Reply by MrViSiOn** [reply]
Question is simple: how can you select a date and time in this control with plain js?

**Reply by ghost** [reply]
as I said with the right datetime format, here is an example https://codepen.io/Niickles/pen/ZEJKEGX

**Reply by MrViSiOn** [reply]
Thank you so much.

I will try as You said,
